imports/plugins/core/checkout/client/components/completedOrder.js

```diff
@@ -34,213 +34,212 @@ class CompletedOrder extends React.Component {
       isProfilePage: props.isProfilePage,
       isCancelled: false
     };
-    this.cancelOrder = this.cancelOrder.bind(this);
-  }
-  cancelOrder(event) {
-    event.preventDefault();
-    Alerts.alert({
-      title: "Cancel Order",
-      type: "question",
-      text: "You're about to cancel this order",
-      showCancelButton: true
-    }, (isConfirmed) => {
-      if (isConfirmed) {
-        const { order } = this.props;
-        const { email } = order;
-        const amount = order.billing[0].invoice.total;
-        const transaction = {
-          amount: Number(amount),
-          to: email,
-          from: email,
-          transactionType: "credit"
-        };
-        Meteor.call("orders/cancelOrder", order, true);
-        Meteor.call("wallet/update-balance",
-          transaction,
-          (err, res) => {
-            if (res === 2) {
-              Alerts.toast(`No user with email ${email}`, "error");
-            } else if (res === 1) {
-              Alerts.toast("Order successfully cancelled", "success");
-              this.setState({
-                isCancelled: true
-              });
-            } else {
-              Alerts.toast("An error occured, please try again", "error");
-            }
-          }
-        );
-      }
-    });
   }
+   cancelOrder = event => {
```

Comment:
Move the `cancelOrder`, and `render` methods back by 1 space. The spacing is off from here onwards - the `cancelOrder` should be aligned with the curly brace in line 37